README.md

```diff
@@ -23,31 +27,117 @@ const job = await client.createReport({
   endTime: new Date().toISOString(),
   browserName: "Chrome",
   browserVersion: "105",
-  framework: "playwright",
+  framework: "playwright", // alternatives: cypress, playwright, testcafe
   frameworkVersion: "1.25.0",
   platformName: "Windows 11"
 });
 console.log(job.id); // the job ID
 console.log(job.url); // the full URL of the job
 ```
 
-## Upload Assets
+### Upload log for JavaScript Batch Frameworks
+
+The minimum `sauce-test-report.json` file to fulfill the web UI:
```

Comment:
"Fulfill the web UI" is a bit terse and awkward. Can you describe a bit more clearly what the file is for?

And the phrasing `The minimum 'sauce-test-report.json' file` makes it sound like the contents of the snippet verbatim below is the "minimum" report required. But I'm guessing what you mean is at a minimum, a file called `sauce-test-report.json` is required?

Should we then also refer to the [sauce-json-reporter](https://github.com/saucelabs/sauce-json-reporter-js) lib for generating the reports?